Find a functionally equivalent code:android.service.dreams.DreamService.onCreate:COMMENT
Method Modifier: public      
Comment:/**
 * Called when this Dream is constructed.
 */

@@ -1,5 +1,5 @@
 {
     if (mDebug)
-        Slog.v(TAG, "onCreate() on thread " + Thread.currentThread().getId());
+        Slog.v(TAG, "onCreate()");
     super.onCreate();
 }

